mongo_connector/util.py

```diff
@@ -32,7 +32,12 @@ def wrapped(*args, **kwargs):
                 return f(*args, **kwargs)
             except:
                 exc_type, exc_value, exc_tb = sys.exc_info()
-                new_type = mapping.get(exc_type)
+                new_type = None
+                for src_type, dest_type in mapping.iteritems():
```

Comment:
`iteritems` doesn't exist in Python 3, so iterate the keys instead:

    for src_type in mapping:
       dest_type = mapping[src_type]
       if issubclass(exc_type, src_type):
           ...